ftlib/consensus/gossip/__init__.py

```diff
@@ -0,0 +1,110 @@
+__version__ = '0.0.1'
+
+import time
+import logging
+from ctypes import *
+from ..basic import BasicConsensus
+from ..consensus_status import ConsensusStatus
+
+
+class Gossip(BasicConsensus):
```

Comment:
I think we could add some comments to let users know that we are using hashicorp memberlist now.

Reply:
If the user wants to use the gossip implementation, should he/she install the dynamic lib?

Reply:
comments added.
if a user want to use this gossip implementation, he/she needs to install the dynamic lib.